packages/react-dom/src/shared/CSSProperty.js

```diff
@@ -34,11 +37,19 @@ export const isUnitlessNumber = {
   gridColumnSpan: true,
   gridColumnStart: true,
   fontWeight: true,
+  initialLetter: true,
   lineClamp: true,
   lineHeight: true,
+  maskBorder: true,
+  maskBorderOutset: true,
+  maskBorderSlice: true,
+  maskBorderWidth: true,
+  maxLines: true,
```

Comment:
This property isn't standardize and *I think* has been superseded by `line-clamp`? So maybe we shouldn't add it here?

Reply:
That’s right, I just wanted to add any property with a status of standard or experimental.